### ISSUE TITLE
Still under development?

### ISSUE THREAD

**Issue by confile** [tothenew/Grails-Mail-Watcher-Plugin#1]
Is this plugin still under development?

**Comment by hiteshBhatia** [comment]
not really, but I might be able to get you in contact with the guys who upgraded this for plugin to use it in their project.

**Comment by drmichaelgorski** [comment]
Can I read only unread mails with this plugin?

**Comment by drmichaelgorski** [comment]
Could you please update the plugin for version Grails 2.3.x
